bundles/org.openhab.binding.zway/README.md

```diff
@@ -188,6 +188,3 @@ Because textual configuration isn't useful, follow the instructions in the [Gett
 -   Z-Way device types (especially the probe types) supported by openHAB channels with detailed information (scale types and so on) are not complete.
 -   Configuration of the Z-Wave network by the binding is currently not possible (physical device configuration)
 -   Only polling is available. Further versions will contain other mechanisms under usage of the WebSocket implementation of Z-Way or MQTT.
-
-<br>
-![BMWi](./doc/BMWi_4C_Gef_en.jpg)
```

Comment:
Was this image present? if so should we remove it?

Reply:
I was wondering what kind of BMW car was showing but I didn't find any when I looked at the [README](https://github.com/openhab/openhab2-addons/blob/master/bundles/org.openhab.binding.zway/README.md) at GitHub. So I removed it. But now I see there is a [BMWi_4C_Gef_en.jpg](https://github.com/openhab/openhab2-addons/blob/master/bundles/org.openhab.binding.zway/doc/BMWi_4C_Gef_en.jpg) image in the doc dir. Apparently it was supported by a German ministry. So it might be better to fix the image link. :+1: